include/data/json.h

```diff
@@ -222,6 +223,68 @@ typedef int (*json_append_bytes_t)(const char *bytes, size_t len,
 		}, \
 	}
 
+/**
+ * @brief Helper macro to declare a descriptor for an array of primitives
+ *        which is inside an array of objects.
+ *
+ * @param struct_ Struct packing the values
+ *
```

Comment:
Spaces between parameters were removed in #37939, if you could remove for consistency that would be greatly appreciated.  

Apologies for all the recent changes in the code base, the library was orphaned and we've been trying to get it back up and going, so there has been a lot of PRs. This fix is greatly appreciated